### ISSUE TITLE
glutin::Event::Closed doesn't work on OS X

### ISSUE THREAD

**Issue by InfiniteDarkness** [tomaka/glutin#257]
For some reason, the window doesn't close if I click on the red dot. It does recognise other events, but not this one:

```rust
extern crate glutin;

fn main()
{
	let window = glutin::Window::new().unwrap();

	unsafe { window.make_current() };

	'main: loop
	{
		for event in window.poll_events()
		{
			match event
			{
				glutin::Event::Closed => break 'main,
				_ => ()
			}
		}
	}
}
```

**Comment by bobakker** [comment]
I guess this can be closed now.

**Comment by tomaka** [comment]
Fixed with #357
